File: Python/program62.py
class Factor:
    _No = 0
    
    def __init__(self, iNo):
        self._No = iNo
           
    def SumFactor(self):
        sum = 0
        for iCnt in range(1,(self._No // 2) + 1,1):
            if((self._No % iCnt) == 0): 
                sum = sum + iCnt

        return  sum
           
 
    
def main(): 
    
    iValue = int (input("Enter number : "))
        
    sobj = Factor(iValue)
    iRet = sobj.SumFactor()
    
    print("Summation of factor is ",iRet)

if __name__ == "__main__":
    main()

File: C/program62.c
#include<stdio.h>


void DisplayNonFactor(int iNo)
{
    if(iNo < 0)
    {
        iNo = - iNo;
    }

    int iCnt = 0;

    printf("Fcators are :\n");
    for(iCnt = 1; iCnt < iNo; iCnt++)
    {
        if((iNo % iCnt) != 0)
        {
            printf("%d\n",iCnt);
        }
    }
}

int main()
{
    int iValue = 0;
   
    printf("Enter number :");
    scanf("%d",&iValue);

    DisplayNonFactor(iValue);

    return 0;
}